### PR TITLE
fix: Remove bin packages from arch

### PR DESCRIPTION
**Pull request by Lutitious**
Removes the bin packages from the arch linux section as they are currently broken.

### CODE REVIEW

**Comment by Scrumplex** [comment]
Isn't the qt5-bin fine?

**Comment by Lutitious** [comment]
> Isn't the qt5-bin fine?

oh right yeah ill change that

**Comment by Lutitious** [comment]
Alright i think thats all good then

File: src/download/linux.md [hunk 1]
```diff
@@ -65,8 +65,6 @@ If you want to use Qt 5 to build the packages instead:
 ```bash
 # stable Qt 5 source package:
 yay -S prismlauncher-qt5
-# stable Qt 5 binary package:
```
Comment (DioEgizio) [inline src/download/linux.md]: Qt5 bin still works

Comment (Lutitious) [reply src/download/linux.md]: > Qt5 bin still works

Fixed

**Review by DioEgizio — APPROVED** [review]

**Pull request merged**